querybook/webapp/components/ListDataDoc/components/LastRun.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import moment from 'moment';
+
+export const LastRun: React.FunctionComponent<{ createdAt: number }> = ({
+    createdAt,
+}) => {
+    return createdAt ? (
+        <div
+            title={`${moment
+                .unix(createdAt)
+                .utc()
+                .format('MMM D, H:mma')}, ${moment
+                .unix(createdAt)
+                .utc()
+                .fromNow()}`}
+        >
+            {moment.unix(createdAt).utc().format('MMM D, H:mma')}
```

Comment:
use generateFormattedDate

Reply:
also no need to run the function twice, for generateFormattedDate, assign it to a variable